.circleci/config.yml

```diff
@@ -5,32 +5,26 @@ jobs:
       - image: python:3.6
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - pip-{{ checksum "requirements.txt" }}
-      - run:
-          name: Install all Python requirements
-          working_directory: ~/project/
-          command: |
-              pip install --progress-bar off --upgrade pip setuptools
-              pip install --progress-bar off -r requirements.txt
-      - save_cache:
-          key: pip-{{ checksum "requirements.txt" }}
-          paths:
-            - ~/.cache/pip
       - run:
           name: Build source distribution and install package from it
+          working_directory: ~/project/
           # Ensure we can build a source distribution that can correctly install.
           # "python setup.py sdist" creates dist/wikipron-x.y.z.tar.gz
           command: |
+              pip install --progress-bar off --upgrade pip setuptools
               python setup.py sdist
               pip install dist/`ls dist/ | grep .tar.gz`
```

Comment:
The wikipron installation test must happen in a clean env without the third-party python packages present, or else we could run into a similar import bug again.

Removed environment cache, as building the env isn't expensive for a super light package like wikipron.

Reply:
SGTM.